src/ElectricPokemon.java

```diff
@@ -0,0 +1,54 @@
+public class ElectricPokemon extends Pokemon{
+    //variables
+    private String type;
+    private String weakTo;
+    private String strongAgainst;
+
+    //constructor
+    public ElectricPokemon(String name, int level, int hp) {
+        super(name, level, hp);
+        this.type = "Electric";
+        this.weakTo = "Grass";
+        this.strongAgainst = "Water";
+    }
+
+    //methods / unique skills
+    public void thunderbolt() {
+        System.out.println(name + " uses Thunderbolt!");
+    }
+
+    public void voltTackle() {
+        System.out.println(name + " uses VoltTackle!");
+    }
+
+    // override
```

Comment:
Dit commentaar is volledig overbodig, want er staat ook al `@Override`. Probeer alleen commentaar in je code te zetten als het wat bijdraagt aan het begrip van je code.